src/minisweagent/agents/interactive.py

```diff
@@ -35,8 +35,8 @@ class InteractiveAgentConfig(AgentConfig):
 class InteractiveAgent(DefaultAgent):
     _MODE_COMMANDS_MAPPING = {"/u": "human", "/c": "confirm", "/y": "yolo"}
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, config_class=InteractiveAgentConfig, **kwargs)
+    def __init__(self, *args, config_class=InteractiveAgentConfig, **kwargs):
```

Comment:
Also happy to add this in a separate PR. Maybe add a `*` before `config_class` to mark it keyword-only